app/page.tsx

```diff
@@ -3,8 +3,8 @@ import Link from "next/link";
 export default function Index() {
   return (
     <div>
-      <Link href="/api/download/secret.json">
-        <a>Download secret file</a>
+      <Link prefetch={false} href="/api/download/secret.json">
+        Download secret file
       </Link>
```

Comment:
**Suggestion:** Add 'as' prop to 'Link' component to provide a more user-friendly URL in the browser's address bar.
```suggestion
      <Link prefetch={false} href="/api/download/secret.json" as="/download/secret">
        Download secret file
      </Link>
```